kimchi/src/folding/test.rs

```diff
@@ -0,0 +1,30 @@
+use crate::folding::expressions::{extract_terms, FoldingColumn, FoldingExp};
+
+#[test]
+fn test_term_separation() {
```

Comment:
What does the test do exactly?

Reply:
actually, it doesn't test anything, I just used it to see the output, I'll make it non default.